pkg/controller/replicaset/replica_set.go

```diff
@@ -485,22 +480,15 @@ func (rsc *ReplicaSetController) manageReplicas(filteredPods []*v1.Pod, rs *exte
 		for i := 0; i < diff; i++ {
 			go func() {
 				defer wg.Done()
-				var err error
-
-				if rsc.garbageCollectorEnabled {
-					var trueVar = true
-					controllerRef := &metav1.OwnerReference{
-						APIVersion: getRSKind().GroupVersion().String(),
-						Kind:       getRSKind().Kind,
-						Name:       rs.Name,
-						UID:        rs.UID,
-						Controller: &trueVar,
-					}
-					err = rsc.podControl.CreatePodsWithControllerRef(rs.Namespace, &rs.Spec.Template, rs, controllerRef)
-				} else {
-					err = rsc.podControl.CreatePods(rs.Namespace, &rs.Spec.Template, rs)
+				isController := true
+				controllerRef := &metav1.OwnerReference{
+					APIVersion: getRSKind().GroupVersion().String(),
+					Kind:       getRSKind().Kind,
+					Name:       rs.Name,
+					UID:        rs.UID,
+					Controller: &isController,
```

Comment:
This part of my PR is going to disappear anyway because someone else did this in another PR and I'm going to rebase onto that after it merges.

Aside from that, I'm not sure it's worth adding a dependency on another package just to get the Go equivalent of `new bool(true)`. I think it's clear enough that we are creating a variable just above and providing a pointer to that variable.